build/migrate-wsl-from-v020.ps1

```diff
@@ -0,0 +1,22 @@
+# PowerShell script to ensure that if there's a 
+# This script returns 0 if a successful migration was carried out, 100 if no migration is necessary
+
+$parentKey = "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Lxss"
+$guids = Get-ChildItem -Path $parentKey | Select-Object -ExpandProperty Name
+foreach ($guid in $guids) {
+   $prop = Get-ItemProperty -Path ("Registry::" + $guid)
+   if ($prop.DistributionName -eq "k3s") {
+     # Need to shut it down. Couldn't get this to work:
+     # wsl --list --running | select-string -Pattern k3s
```

Comment:
FWIW, it looks like the issue is that `wsl --list` outputs in UTF-16-LE, and PowerShell doesn't expect that, instead splitting as if it's UTF-8 (so some strings have a leading null byte), which then confuses `Select-String`.

That's okay, though, always terminating sounds better.

Reply:
The whole time I was writing this I was wondering about encodings, but that was all on the registry side, and was happy to see that it worked ok. Since our old name and new names are both pure ascii, I didn't see a problem. But this explains why only `.` was matching.